### DISCUSSION TITLE
“Users/my_username/.config” exists and it is not owned by the current user

### DISCUSSION THREAD

**Discussion by lint888** [podman-desktop/podman-desktop#8496] [Q&A]
Hello everyone,

I'm trying to install podman desktop on macos Sonoma 14.6.1, m3 max chip.
I successfully installed podman desktop and I need to install Podman because "Podman needs to be set up". I ran the installation process and I got the following error message 
**“Users/my_username/.config” exists and it is not owned by the current user"** 
when I run
**"podman machine init"**

I considered using sudo chown command but this is my working laptop and my college suggested me not to use the sudo chown command. Is there other solutions ?

Thank you in advance!

Best,

**Comment by feO2x** [comment]
I got my first MacBook Pro yesterday and ran into the same issue. Before installing podman, I installed miniconda3 for python environment management. This created the `~/.config` directory which belonged to the root user (in it, you should find `~/.config/fish/config.fish` file which registers conda with the fish shell).

I transferred ownership of the directory to my user by executing the following command:

```zsh
sudo chown -R <your-user-name> ~/.config
```

...where you need to replace `<your-user-name>` with the actual name of your user. Afterwards, I was able to run `pod machine init`.

I think this shouldn't influence future miniconda3 updates as it should have access via the root user - but I'll report back if I face any issues.

**Reply by feO2x** [reply]
To be a little bit more specific: I ran `ls -ld ~/.config` to identify which user the '.config' folder actually belonged to (third column of the output).

**Reply by saurabh180387** [reply]
Thanks @feO2x , this works for me as well, since after ownership transfer , "init" is successful

**Reply by dnielsen** [reply]
Same. This worked for me too.
